pkg/apis/rio.cattle.io/v1/service_types.go

```diff
@@ -359,11 +362,29 @@ type ImageBuild struct {
 	// Specify the name Of the Dockerfile in the Repo. Defaults to `Dockerfile`.
 	DockerFile string `json:"dockerFile,omitempty"`
 
+	// Specify build context
+	DockerContext string `json:"dockerContext,omitempty"`
```

Comment:
nit: I feel this should be called `BuildContext`, as `docker context` means a different thing (like kubectl context, since Docker 19.03)